apps/website/docs/infrastructure/migrate-opex.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 5
+---
+
+# Updating Opex Dashboards and Alerts with new Application Gateway
+
+Opex dashboards and alerts are used to monitor the performance and health of
+Azure Application Gateways. However, when a new Application Gateway is created,
+the existing dashboards and alerts do not automatically update to replace the
+old Application Gateway with the new one. This may lead to unused alerts and
+empty dashboards.
+
+In order to replace the Application Gateway instance, dashboards and alerts must
+be recreated, as it is not possible to update the `Scope` property of existing
+alerts.
+
+This guide explains how to safely delete and recreate Opex dashboards and alerts
+when migrating to a new Azure Application Gateway.
+
+## Steps to Recreate Dashboards and Alerts
+
+Unlike any other Terraform resource, Opex dashboards and alerts are created
+using a custom Python application, and therefore Terraform source code is not
+available to the user. Therefore, the quickest way to delete programmatically
+existing dashboards and alerts is to create an empty Terraform configuration and
+apply it. This action will remove all existing dashboards and alerts, allowing
+you to recreate them with the new Application Gateway.
+
+1. Create a new directory for an empty Terraform configuration:
+
+```bash
+mkdir empty-terraform-config
+cd empty-terraform-config
+```
+
+2. Create a new file named `main.tf` with the following content:
+
+```hcl
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 4.0"
+    }
+  }
+
+  backend "azurerm" {
+    resource_group_name  = "<your-resource-group-name>"
+    storage_account_name = "<your-storage-account-name>"
+    container_name       = "<your-container-name>"
+    key                  = "<your-terraform-state-file-name>"
```

Comment:
suggestion: I'd make it clear here that these values must refer to the opex tf state only

Reply:
I see, but to ensure clarity and prevent reader confusion, it's always best to provide contextual hints precisely when the information is relevant. Otherwise, readers may be left wondering "how" until they reach a subsequent paragraph.